src/main/java/io/rpg/view/LocationView.java

```diff
@@ -62,6 +70,11 @@ public static LocationView fromConfig(LocationConfig config) throws IOException
     System.out.println(config.getBackgroundPath());
     view.getViewModel().setBackground(new Image(config.getBackgroundPath()));
     // todo: na podstawie configu ustawić pola korzystając z view modelu
+    List<GameObjectConfig> objectConfigs = config.getObjects();
+    for(GameObjectConfig objectConfig: objectConfigs){
+      GameObjectView goview = new GameObjectView(Path.of(objectConfig.getAssetPath()), objectConfig.getPosition());
+      viewModel.addChild(goview);
+    }
```

Comment:
Jest spoko, ale można przemyśleć tworzenie obiektów `GameObjectView` już w `Initializer`. Wyciągnięcie tego tam i tak będzie potrzebne, zdaje się, bo będzie trzeba zarejestrować view do modelu (czyli każdy `GameObjectView` dodać jako observera do odpowiadającego mu `GameObject`). Prawdopodobnie zrobię [w tym PR](https://github.com/RPG-IO/rpg-io/pull/16), ale jeszcze nie wiem na kiedy to będzie gotowe.